teamcity/tox.ini

```diff
@@ -2,19 +2,29 @@
 minversion = 2.0
 basepython = py27
 envlist =
+    unit
     teamcity
     flake8
 
 [testenv]
 platform = linux|darwin|win32
 
-[testenv:teamcity]
+[common]
```

Comment:
no need for a `common` section actually, you can put the common parts under `testenv` and override when needed in `testenv:foo`